src/Admin/BaseMediaAdmin.php

```diff
@@ -112,7 +112,7 @@ public function configurePersistentParameters(): array
         ]);
     }
 
-    public function alterNewInstance(object $object): void
+    final public function alterNewInstance(object $object): void
```

Comment:
not sure about this one too.

Reply:
but this can be protected

Reply:
According to https://github.com/sonata-project/SonataMediaBundle/pull/2194#discussion_r740782622, we should mark everything as final by default

The scope of the method comes from the `AdminInterface`

Reply:
`alterNewInstance` does not come from the AdminInterface.
It's already not final in the AbstractAdmin because it's an extension point.
That's why we should not open the visibility, it's protected in the AbstarctAdmin and should stay it.
Extension point of the AbstractAdmin shouldn't be final in AbstractMediaAdmin because it's still an extension point.

In the same way configureListField wasn't final.

Reply:
Yes, I was not sure about this one, I will revert it.

Reply:
Revert the final keyword.